packages/core/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -20,11 +20,13 @@ import LinkIcon from '@material-ui/icons/Link';
 import { Link as RouterLink } from '../Link';
 
 export type IconLinkVerticalProps = {
```

Comment:
`action` is being removed from props, so this would be a breaking change, i.e. needs a `minor` bump of `@backstage/core`.

Reply:
Components are stability `1`, so we'd also need a bit of a guide in the changeset for people that are currently using the `action` prop and how they would migrate their code.